UnityProject/Assets/Scripts/Health/Objects/Armor.cs

```diff
@@ -30,6 +31,14 @@ public class Armor
 
 	public bool StunImmunity = false;
 
+
+	[MinMaxSlider(0.00f, 50000.0f)]
+	public Vector2 TemperatureProtectionInK = new Vector2( 268.15f, 313.15f);
```

Comment:
I don't think these need to be set outside the class, do they?

Reply:
I think they need to be got from outside the class but apart from that it's good

Reply:
In that case, feel like it would be preferred to use { get; private set; } right?

Reply:
editor doesn't understand it if you do that

Reply:
[field: SerializedField]

We have talked about it before

Reply:
![image](https://user-images.githubusercontent.com/27101670/209545871-e34f447c-5d28-4237-bdcf-87b52e56b1fb.png)

Reply:
[field: SerializeField, MinMaxSlider(0.00f, 50000.0f)], you can put field before any property to have the attribute treat the property as if it was a field

Reply:
![Capture](https://user-images.githubusercontent.com/48405920/209547222-de372eae-fa67-4514-baea-61bd5760f362.PNG)
Like so, (ExplosiveBounty.cs)

Reply:
for some reason it loses the serialisation reference, and I can't be bothered doing the numbers again since it was majority of the time on the PR